Cargo.toml

```diff
@@ -31,6 +38,9 @@ escargot = "0.5.7"
 predicates = "3.0.3"
 shlex = "1.1.0"
 
+[features]
+default = ["bytecount/runtime-dispatch-simd"]
```

Comment:
This I also just mimicked as it looked appropriate to do so but don't know exactly what this invokes